File: 0485-max-consecutive-ones/0485-max-consecutive-ones.py
class Solution(object):
    def findMaxConsecutiveOnes(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        c=0 
        b=0
        while c!=len(nums):
            a=0
            for j in range (c, len(nums)):
                if (nums[j]==1):
                    a+=1
                    c+=1
                else:
                    c+=1
                    break
            if (a>b):
                b=a
        return b
        

File: 0485-max-consecutive-ones/0485-max-consecutive-ones.java
class Solution {
    public int findMaxConsecutiveOnes(int[] nums) {
        int max = 0 ;
        int count = 0 ;
        int n = nums.length; 
        for(int i = 0 ; i<n ; i++){
            // count = 0 ;
            if (nums[i] == 1){
                count +=1;

            }
            else{
                max= Math.max(count , max );
                count = 0;
            }

        }
        max= Math.max(count , max );
        
        return max;
    }
    
}